examples/pytorch_sample/pytorchjob.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: "kubeflow.org/v1alpha1"
+kind: "PyTorchJob"
+metadata:
+  name: "pytorch-smoke"
+spec:
+  backend: "tcp"
+  masterPort: "23456"
+  replicaSpecs:
+    - replicas: 1
+      replicaType: MASTER
+      template:
+        spec:
+          containers:
+          - image: gcr.io/refined-byte-127219/pytorch-smoke-test:1.0
```

Comment:
You aren't actually using the container you are building.  I think it should be `gcr.io/kubeflow-ci/pytorch-smoke-test:1.0`